updates/testing.json

```diff
@@ -93,6 +93,16 @@
           "start_percentage": 0.0
         }
       }
```

Comment:
It's a bit confusing to me to have two overlapping rollouts. Should we just cancel the one above by setting the start percentage to `1.0`?

Reply:
same for `next` too

Reply:
No, I think the overlapping rollouts are correct.  We shouldn't stop the OpenSSL fix rollout just because some nodes haven't updated to the new releases yet.